react/react1/week2/todo-list-app/src/TodoList.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+import { todos } from './todos'
+
+const TodoList = () => {
+  //   mark a To-do as done
+  const [todosState, setTodosState] = useState(todos)
+  console.log(todosState)
+  const setTodoAsCompleted = (id) => {
+    let newTodosState = todosState.map((todo) => {
+      if (id === todo.id) {
+        if (todo.isComplited === false) {
+          todo.isComplited = true
+          return todo
+        } else {
+          todo.isComplited = false
+          return todo
+        }
+      }
```

Comment:
Instead, you can use like
`const setTodoCompleted = (id) => {
todos.map((todo)=>{
if(id === todo.id){
todo.isCompleted=true;}
})
}`
would be enough, if you already defined isCompleted.

Reply:
I was try to make a todo toggle able. Like if user click on item once, can see status done. If press once more it changes  back.